app/org/sagebionetworks/bridge/services/SurveyService.java

```diff
@@ -291,6 +295,20 @@ public Survey getSurveyMostRecentlyPublishedVersion(StudyIdentifier studyIdentif
 
         return surveyDao.getAllSurveysMostRecentVersion(studyIdentifier, includeDeleted);
```

Comment:
All the list-based methods use study identifier to query for the surveys, rather than GUID, so they will always matched the supplied study identifier. (In the controller we get this study identifier from the user's session).